### ISSUE TITLE
Elementor Loop Grid Archive Page Not Showing Timer

### ISSUE THREAD

**Issue by creativejesh** [jamesgol/woocommerce-cart-stock-reducer#97]
Ive designed a new product archive template for my shop. Im using the new loop grid setup with an Ajax filter and its running very well. However, Im now not getting the timer like I did previously.

**Comment by jamesgol** [comment]
Sorry for the delay.  I'm going to have to dig into this.  Is the new loop grid setup a new WooCommerce thing or something specific to your theme?

**Comment by creativejesh** [comment]
Im sorry I should have clarified. Loop grid is a new elementor feature. Using elementor Hello theme. I had a buddy dig into it a bit and couldnt figure it out so I figured id just wait until I heard from you. I really appreciate you looking into it. Be happy to put some dono down to help get it fixed.

**Comment by jamesgol** [comment]
Is this feature available in the free Elementor version?  

```
The Loop Grid is a widget but acts a little differently from most widgets. In most cases, users drag the widget onto the canvas and then edit them in the panel. With the Loop Grid, however, after you drag the widget onto the canvas, you’ll be asked to create a template. That’s because when you build a loop, you’re creating a template that will be used to display a listing of various items in a uniform way. 
```

What does your template look like?

**Comment by creativejesh** [comment]
I don't believe it's in the free version. My template is just a block for products that are displayed in a list style. I went this route because I want a clean way to have a list format product archive or shop. Unlimited addons has a widget that does this but its clunky and also breaks the timer notice. See screenshots of my template below.
<img width="1454" alt="Untitled 7" src="https://github.com/jamesgol/woocommerce-cart-stock-reducer/assets/142461803/545ca1f1-1941-484f-8837-033af821be47">
<img width="1149" alt="Untitled 8" src="https://github.com/jamesgol/woocommerce-cart-stock-reducer/assets/142461803/74dec280-cb9f-4d29-854f-78f9e26180e9">

Thanks again!
